src/parser.rs

```diff
@@ -451,7 +451,7 @@ impl_rdp! {
                 Ok(Some(Node::VariableBlock(Box::new(exp?))))
             },
             (_: raw_tag, &body: raw_text, _: endraw_tag) => {
-                Ok(Some(Node::Raw(body.to_string())))
+                Ok(Some(Node::Raw(body.trim().to_string())))
```

Comment:
it would be better to keep the trimming in the renderer so we only have one place to look at